scripts/monorepo/src/isConvergedPackage.js

```diff
@@ -46,4 +88,5 @@ function isNightlyVersion(version) {
   return semver.major(version) === 0 && semver.minor(version) === 0 && semver.patch(version) === 0;
 }
 
-module.exports = isConvergedPackage;
+exports.isConvergedPackage = isConvergedPackage;
+exports.shipsAMD = shipsAMD;
```

Comment:
does this util really belong in `isConvergedPackage`?

Reply:
yeah it doesn't matter from public API POV. I added it here as I was moving parts and it was related to the isConverged check in preset.